pom.xml

```diff
@@ -64,7 +64,7 @@
           <!-- Sets the VM argument line used when unit tests are run. -->
           <argLine>${surefireArgLine}</argLine>
           <!-- Skips unit tests if the value of skip.unit.tests property is true -->
-          <skipTests>${skip.unit.tests}</skipTests>
+          <skipTests>true</skipTests>
```

Comment:
how come this was hard-coded to a value instead of using the property?

Reply:
i fixed it in second commit